src/Models/code/Transformers/CodeTransformer.ts

```diff
@@ -0,0 +1,28 @@
+import Code from '../Code';
+import CodeInterface from '../../../Interfaces/code/Code';
+import BaseTransformer from '../../../BaseTransformer';
+import TypeInterface from '../../../Interfaces/code/Type';
+import TypeTransformer from './TypeTransformer';
+
+export default class CodeTransformer extends BaseTransformer {
+    /**
+     * Map an order response
```

Comment:
```suggestion
     * Map a code response
```

Reply:
Let's just delete this comment altogether I think

Reply:
ok fine with me :)